### PR TITLE
Project finished

### PR DESCRIPTION
**Pull request by webmarkyn**
HTML&JS tic tac toe is finished.
Open index.html in your browser, enter your name and name of your friend, then press "start" button to start the game. "Reset" to reload the game.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
+/* eslint-disable no-alert */
```
Comment (bellom) [inline js/main.js]: **Can all inline eslint rule be removed, please define eslint rule in the eslint file.**

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+/* eslint-disable no-undef */
+
+// player factory for create player objects
```
Comment (bellom) [inline js/main.js:1]: **Kindly remove commented code, you can always use git history to access previous code right :slightly_smiling_face:**

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+const playerFactory = (id, name, token) => ({
+  id,
+  name,
+  token,
+});
+
+let counter = 0;
```
Comment (bellom) [inline js/main.js]: we shouldn't use global variable right 😉 
It doesn't use `global` code - encapsulate logic inside modules or factories ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/global_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript))

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 
#### Awesome job so far!! :+1: 
#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1:, just few issues left to work on............ :slightly_smiling_face: 

Below are part of the checklist for this milestone, kindly work on these few things listed below;

1. Your PR title should always be the features added or the main goal of the project.
You used Factory Functions and The Module. You can always include that in your PR title.

2. Little bug;
You're not checking for all the winning conditions; after player `x` already won, player `o` can still play, screenshot below;
image

3. Once we have a winner we shouldn't be allow to continue with the game, but in this case we can;
image
**The loom recording link below explained the issue more better;**
https://www.loom.com/share/e5dba37b3c2f41bd80c8bfa6b5546490


**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: js/main.js (hunk 1 above)
Comment (oscardelalanza) [reply js/main.js]: done

File: js/main.js (hunk 2 above)
Comment (oscardelalanza) [reply js/main.js:1]: done

File: js/main.js (hunk 3 above)
Comment (oscardelalanza) [reply js/main.js]: done

**Comment by oscardelalanza** [comment]
### List of Changes

- `Commented code` was removed from the project

- Linter rules in the `main.js` moved to `eslint.conf`

- `Validation Bug` fixed avoiding empty spaces on the board

**Review by bellom — APPROVED** [review]
## Hi Team :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Goodluck in the next project :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**